sdk/python/kfp/dsl/_container_op.py

```diff
@@ -17,11 +17,125 @@
 from . import _pipeline_param
 from ._pipeline_param import _extract_pipelineparams
 import re
-from typing import Dict
-
+from typing import Dict, List
+from abc import ABCMeta, abstractmethod
+from . import _types
+
+class BaseMeta(object):
+  __metaclass__ = ABCMeta
```

Comment:
I think this is Py2 way. In Py3, it's better to do 

BaseMeta(metaclass=ABCMeta)

https://docs.python.org/3/reference/datamodel.html#metaclasses

Reply:
I do not quite understand why are we using metaclass here.